specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -3626,25 +3626,25 @@
         }
       }
     },
-    "ValidateAddressResponse": {    
-    "type":"object",
-    "description" :" Result of the address validation",
-    "properties" : {
-      "status": {
-        "description": "status of the address validation.",
-        "$ref": "#/definitions/AddressValidationStatus"
-      },
-      "suggestedAddresses": {
-        "description": "list of suggested addresses.",
-        "type": "array",
-        "items": {
-          "$ref": "#/definitions/Address"
+    "ValidateAddressResponse": {
+      "type": "object",
+      "description": "Result of the address validation",
```

Comment:
I manually removed the leading space in the description after the automated formatting